www/app/assets/js/app.js

```diff
@@ -2195,20 +2244,20 @@ angular.module('kubernetesApp.components.dashboard')
     var pods = {
       "kind": "PodList",
       "creationTimestamp": null,
-      "selfLink": "/api/v1beta1/pods",
+      "selfLink": "/api/v1beta3/pods",
       "resourceVersion": 166552,
-      "apiVersion": "v1beta1",
+      "apiVersion": "v1beta3",
```

Comment:
As in the other PR, all of these examples are still using the v1beta1 schema. They need to be totally regenerated using v1beta3.